src/App.tsx

```diff
@@ -9,32 +9,53 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
+interface TabType {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type TabListProps = {
+  tab: TabType[],
+};
+
+export const App: React.FC<TabListProps> = () => {
+  const [activeTab, seActiveTab] = useState(tabs[0]);
```

Comment:
Typo in the useState setter name, it should be setActiveTab instead of seActiveTab